### ISSUE TITLE
[Bug] Stretching vs non stretching has a different collision result

### ISSUE THREAD

**Issue by ProPeach** [tsunamayo/Starship-EVO#3381]
### Short Description of the issue:
When building on a child entity close to the parent entity, the collision check comes up with different results if the player stretches a brick vs if they don't stretch it
https://i.imgur.com/P7Ptnyg.mp4

### Steps to reproduce the issue:
Load the blueprint attached
Try and place bricks in the same place as I did in the Imgur video

### Environment
- Starship EVO version: 20w50a
- Graphics Card: RX580

### Additional information:
[Fighter Test.zip](https://github.com/tsunamayo/Starship-EVO/files/5699459/Fighter.Test.zip)
[output_log.txt](https://github.com/tsunamayo/Starship-EVO/files/5699460/output_log.txt)

**Comment by tsunamayo** [comment]
Interesting case, thanks for booking! Cheers

**Comment by Kaiser-Indrasil** [comment]
I've got a new test case.
Given the child entity clipping, apparently things like this are still somehow possible:
![Starship_EVO_xaxndNfSXl](https://user-images.githubusercontent.com/63756057/103188150-3cbe7400-48bf-11eb-90f2-fe44619aca9a.png)
Yet, this is not okay with the game for some reason:
![Starship_EVO_zKuWhJC3Dw](https://user-images.githubusercontent.com/63756057/103188164-4b0c9000-48bf-11eb-8164-d6c7748cf3f3.png)
No stretching involved.
I've noticed the same bug with tetra and corner bricks.
Hepta, on the other hand, seems fine.
